blackmarket/UI/Views/SignUpView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  SignUpView.swift
+//  blackmarket
+//
+//  Created by Daniel Parra on 9/30/22.
+//
+
+import SwiftUI
+
+struct SignUpView: View {
+    
+    @Environment(\.presentationMode) var presentationMode
+    @ObservedObject var viewModel = SignUpViewModel()
+    
+    var body: some View {
+        VStack(spacing: 16.0) {
+            VStack {
+                Image("logo")
+                    .scaledToFit()
+                    .padding(.bottom, 36.0)
+                
+                VStack {
+                    TextFieldView(fieldConfiguration: $viewModel.emailConfiguration)
+                    TextFieldView(fieldConfiguration: $viewModel.nameConfiguration)
+                    TextFieldView(fieldConfiguration: $viewModel.passwordConfiguration)
+                }
+                .padding(.horizontal, 28.0)
+
+                StateButton(action: {
+                    viewModel.logIn()
+                }, title: LocalizedString.SignUpScreen.signUpButtonTitle, isEnabled: viewModel.isValid)
+                .frame(maxHeight: 45)
+                .padding(.horizontal, 28.0)
+                .padding(.bottom, 16.0)
+                
+                Text(viewModel.disclaimerString)
+                    .font(.subheadline)
+                    .multilineTextAlignment(.center)
+                    .tint(Color.ui.link)
+                    .padding(.horizontal, 28.0)
+                    .padding(.bottom, 16.0)
+                
+                Text(viewModel.alreadyHaveAnAccountString)
+                    .font(.subheadline)
+                    .padding(.horizontal, 28.0)
+                    .padding(.bottom, 16.0)
+                    .onTapGesture {
```

Comment:
should we use a button instead?

Reply:
The behavior is different because the button has a onTap highlight animation that is not desired here

Reply:
Ok lets leave this as is for now, in any case we can add styling to the button text directly